### PR TITLE
Use @embroider/test-setup to test the addon against Embroider

### LINKED ISSUE #12: Test against Embroider
We should add a new `ember-try` scenario to test the app against Embroider (`safe` & `optimized`)

See https://github.com/embroider-build/embroider/tree/main/packages/test-setup

### PR DESCRIPTION
**Pull request by moussamew**
This PR aims to add `@embroider/test-setup` lib to tests `ember-prismisc-dom addon` against Embroider app

Closes #12

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (dbendaou) [inline .github/workflows/ci.yml:1]: Could we exclude not related change to Embroider from this MR? Even though it was wrongly formatted, we should try to keep PR as focus as possible on one thing :)

Comment (moussamew) [reply .github/workflows/ci.yml:1]: Yes! It is now back to its original formatting :)

**Review by dbendaou — APPROVED** [review]
Thanks for your first contribution! 🔥

File: config/ember-try.js [hunk 1]
```diff
@@ -74,6 +75,8 @@ module.exports = async function () {
           },
         },
       },
+      embroiderSafe(),
+      embroiderOptimized(),
```
Comment (dbendaou) [inline config/ember-try.js:79]: Actually, we would like to allow these two jobs to fail for now, could you update it?

**Review by dbendaou — CHANGES_REQUESTED** [review]

File: config/ember-try.js [hunk 2]
```diff
@@ -74,6 +75,8 @@ module.exports = async function () {
           },
         },
       },
+      embroiderSafe(),
```
Comment (dbendaou) [inline config/ember-try.js:78]:
```suggestion
      embroiderSafe({allowedToFail: true}),
```
something like this

Comment (dbendaou) [reply config/ember-try.js:78]: see example here https://github.com/storybookjs/ember-cli-storybook/blob/27bf1b433ba86c88fdf8f858b235ebaac6355035/config/ember-try.js#L86

Comment (moussamew) [reply config/ember-try.js]: Thanks! I was trying to get the `API` and check the existing options, seems to be good

**Pull request merged**